client/webserver/webserver.go

```diff
@@ -904,3 +908,13 @@ func folderExists(fp string) bool {
 	stat, err := os.Stat(fp)
 	return err == nil && stat.IsDir()
 }
+
+// chiLogger is an adaptor around dex.Logger so that it can be used seamlessly
+// with chi library.
+type chiLogger struct {
+	logger dex.Logger
+}
+
+func (l *chiLogger) Print(v ...interface{}) {
+	l.logger.Trace(v...)
```

Comment:
The named field is not necessary here.  Also let's be specific in the docs for `chiLogger` since this is an internal type:

```go
// chiLogger is an adaptor around dex.Logger that satisfies
// chi/middleware.LoggerInterface for chi's DefaultLogFormatter.
type chiLogger struct {
	dex.Logger
}

func (l *chiLogger) Print(v ...interface{}) {
	l.Trace(v...)
}
```

Reply:
Applied.